### ISSUE TITLE
Better treasury management 

### ISSUE THREAD

**Issue by july5613** [FantomMenaceDAO/MIPs#8]
# Menace Improvement Proposal (MIP-9)

## tl;dr
- Better treasury management of our multi-sig treasury wallet; put our FTM to work, (do some defi staking or farming?)

## Motivation
A clear and concise description of what the improvement is. What is the pain point, and how does the improvement solve it?
> @acedabook: i'm prob getting ahead of myself w/ this, but we will eventaully have a capital problem e.g. too much of it and will need to be responsible. There are pro DAOs that can do this for other DAOs, but with a big and experienced enough team we can manage
> 
> from https://discord.com/channels/893513538633613392/893513538633613396/900901212558356560

## Scope
- Feel to add more here what you would be proposing to do (@acedabook)
- Maybe just putting our FTM to work

## Owners
List folks who are going to **owners** for the project and be responsible for delivering
- @acedabook, @ftm_ecologist, @tweak896

## Resources
Do you need funding? Do the contributors to making this improvement need to be paid in NFTs or FTM?
- Not at the moment
